src/main/java/io/microprofile/config/Config.java

```diff
@@ -67,21 +63,7 @@
      */
     Boolean getBoolean(String propertyName);
```

Comment:
Couldn't this be a `default` method delegating to `getProperty(propertyName, Boolean.class)`?